test-site/package-lock.json

```diff
@@ -22,7 +22,7 @@
     },
     "..": {
       "name": "@yext/analytics",
-      "version": "0.2.0-beta.5",
+      "version": "0.3.0",
```

Comment:
Should you keep the beta suffix?

Reply:
This was auto-updated according to what the top level package.json have: https://github.com/yext/analytics/blob/main/package.json#L3

Reply:
sgtm!